tools/base/tests/test_runner.py

```diff
@@ -54,9 +54,18 @@ def run(self, *args, **kwargs):
                 raise self.raises("AN ERROR OCCURRED")
             return result
 
+        @runner.catches(errors)
+        async def run_async(self, *args, **kwargs):
+            result = self._runner(*args, **kwargs)
+            if self.raises:
+                raise self.raises("AN ERROR OCCURRED")
```

Comment:
Is this the best we can do? No additional contextual information? Just wondering, as it doesn't seem super actionable on its own.

Reply:
comment added at the top of the dummy class